src/cplus_plugin/definitions/defaults.py

```diff
@@ -23,7 +23,21 @@
 )
 
 OPTIONS_TITLE = "CPLUS"  # Title in the QGIS settings
+REPORT_OPTIONS_TITLE = "Reporting"
+LOG_OPTIONS_TITLE = "Logs"
 ICON_PATH = ":/plugins/cplus_plugin/icon.svg"
+REPORT_SETTINGS_ICON_PATH = str(
+    os.path.normpath(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        + "/icons/report_settings.svg"
+    )
+)
+LOG_SETTINGS_ICON_PATH = str(
```

Comment:
Here we can use the "Result viewer" icon available under the QGIS  "Processing" menu actions as the icon for the log settings entry.

Reply:
Done. Adapted `processingResult.svg`.